sdk/storage/src/core/clients/storage_account_client.rs

```diff
@@ -58,6 +59,50 @@ pub struct StorageAccountClient {
     account: String,
 }
 
+/// Options for specifying how a Cosmos client will behave
+#[derive(Debug, Clone, Default)]
+pub struct StorageAccountOptions {
+    options: ClientOptions,
+    // custom_dns_suffix: Option<String>,
+}
+
+impl StorageAccountOptions {
+    /// Create new options
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    #[cfg(feature = "mock_transport_framework")]
+    /// Create new options with a given transaction name
+    pub fn new_with_transaction_name(name: String) -> Self {
+        Self {
+            options: ClientOptions::new_with_transaction_name(name.into()),
+        }
+    }
+}
+
+/// Create a Pipeline from CosmosOptions
```

Comment:
This comment is wrong

Reply:
will fix - even though most stuff is heavily inspired by cosmos :)